language/move-prover/tests/sources/functional/mut_ref_accross_modules.move

```diff
@@ -76,22 +76,22 @@ module TestMutRefs {
     }
 
     // For a public function, the data invariant for a mut ref is expected to hold.
-    public fun data_invariant(x: &mut T) {
+    public fun data_invariant(x: &mut T) { // Shaz: This function verifies because resource invariant is assumed on the inout value modeling x
```

Comment:
Is this a comment or an issue we need to fix? If it is an issue please clearly mark a TODO here and state it needs to be fixed. If not I'm wondering what the comment is about. if its a clarification, please remove redundant information ("Shaz") and put the comment before the function. (Which also says something similar?) Same for other cases.

I remember you said that where issues with some tests but I can't find them documented in the PR.

Reply:
All the issues were in these "Shaz:" comments in this one test.  I have improved the comments to explain the issue more precisely.